packages/server/package.json

```diff
@@ -88,6 +88,11 @@
       "import": "./dist/subscription.mjs",
       "require": "./dist/subscription.js",
       "default": "./dist/subscription.js"
+    },
+    "./shared": {
+      "import": "./dist/shared.mjs",
+      "require": "./dist/shared.js",
+      "default": "./dist/shared.js"
```

Comment:
Cc @sachinraja do you think we can throw `observables` and `rpc` into this as well rather than importing them separately?

Then it's slightly easier to create shared utilities, esp considering #2108.

Any modern bundler would tree-shake unusued stuff but I don't know the case for [Fresh](https://fresh.deno.dev/) etc

Reply:
The observables are already imported in the client links so that doesn't matter. Our rpc stuff is not, but that's mostly just types, so it should be fine. Deno/Fresh already imports all of the links unnecessarily because of [this file](https://github.com/trpc/trpc/blob/next/packages/client/src/links/index.ts). This might be harsh but I could also care less about optimizing for non-tree-shaking environments.

Reply:
Wait it should be completely fine, looks like Fresh does do tree-shaking, just at request time: <https://github.com/denoland/fresh/blob/main/src/server/bundle.ts>.

Reply:
How the fuck does it do tree-shaking without a build step? Is it some sort of JIT-buildstep?

Reply:
That sounds so slow

Reply:
Created #2110